osu.Game/Overlays/Music/PlaylistOverlay.cs

```diff
@@ -124,6 +138,10 @@ private void itemSelected(BeatmapSetInfo set)
             playSpecified(set.Beatmaps[0]);
         }
 
+        private void onBeatmapSetAdded(BeatmapSetInfo s) => list.AddBeatmapSet(s);

```

Comment:
as we don't need the dispose/unbind, you can just move these to be inline anonymous delegates